src/verifiers/documentStoreIssued/openAttestationDocumentStoreIssued.ts

```diff
@@ -0,0 +1,24 @@
+import { v2, v3, WrappedDocument } from "@govtechsg/open-attestation";
```

Comment:
Any reason to add this ? I feel this is unnecessary :thinking: 
- can add openAttestationW3CDIDProof in the list of verifiers (index)
- `openAttestationDocumentStoreIssued` wont run if there is no document store (thanks to skip)
- `openAttestationW3CDIDProof` wont run if there is no proof (thanks to skip)

that way we have a result for each verifiers

Reply:
My thought was that having both of those verifiers felt redundant as they are serving the same purpose and we only need one to be successful.  Also there may be cases where a document store and proof block are both included but the document was not issued to the document store. For example when the document stored is used as just the revocation registry for a specific issuer.  This would then fail using the `openAttestationEthereumDocumentStoreIssued` verifier but is in fact valid due to the signed proof block.